app/__init__.py

```diff
@@ -54,14 +53,21 @@ def create_app(config_name):
     # app.register_blueprint(auth_blueprint, url_prefix='/auth')
 
     from .main import main as main_blueprint
-    app.register_blueprint(main_blueprint)
+    from .jobs import jobs as jobs_blueprint
+    from .flowcharts import flowcharts as flowchart_blueprint
 
+    app.register_blueprint(main_blueprint)
+    app.register_blueprint(jobs_blueprint)
+    app.register_blueprint(flowchart_blueprint)
```

Comment:
Each blueprint should use a `url_prefix` to avoid overriding each other.

Reply:
I'm having some trouble understanding how to do this with this with the way the app is set up. In each `view.py` file, the url routes are set up to be prefixed with `/views` because of the way the CoreUI menu behaves. 

For example, in the `index.html` file, for example where the side navigation link is set up, there the following linking to the jobs list page.

```
<li class="nav-item">
              <a class="nav-link" href="jobs_list">
                <i class="nav-icon icon-list"></i> Jobs List</a>
```

The `href` only says 'jobs_list`, but it links to the view defined by the url `views/jobs_list`.

It seems that for the URLs to work with the CoreUI template, they have to have 'views' in the beginning of the URL. Adding a url_prefix (`url_prefix=jobs`) when registering this blueprint would make the URL equivalent to `/jobs/views/jobs_list`, which if I do this, I can view that url, but do not know how to link it in the side menu of the app. It no longer works because the first prefix is not `views`.

Reply:
^ I wasn't able to figure out how to add a url_prefix with the current set-up of the app.